onnx_tf/handlers/backend/dynamic_slice.py

```diff
@@ -0,0 +1,71 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("DynamicSlice")
+@tf_func(tf.slice)
+class DynamicSlice(BackendHandler):
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    tensor_dict = kwargs["tensor_dict"]
+    input_tensor = tensor_dict[node.inputs[0]]
+    starts = tf.cast(tensor_dict[node.inputs[1]], tf.int64)
+    ends = tf.cast(tensor_dict[node.inputs[2]], tf.int64)
```

Comment:
Why do we need `cast` here? ONNX ensures `starts` and `ends` are `int32` or `int64` which is accepted in tf.